server/static/js/device_status/device_status_page_highcharts.js

```diff
@@ -0,0 +1,252 @@
+                tooltip: {
+                    valueDecimals: 1
+                }
+            }],
+        });
+
+        let chartRamUsed = Highcharts.chart('ram_used_kb', {
+            chart: {
+                type: 'line'
+            },
+            title: {
+                text: 'RAM Used kB'
+            },
+            xAxis: {
+                title: {
+                    text: 'Time'
+                },
+                categories: ramUsedTime,
+                tickInterval: 24 * 3600 * 1000
+            },
+            yAxis: {
+                title: {
+                    text: 'Used kB'
+                },
+                max: 512000
+            },
+            legend: {
+              enabled: false
+            },
+            colors: ['#ecf0f1'],
+            series: [{
+                name: 'RAM Used kB ' + deviceName,
+                data: ramUsed,
+                tooltip: {
+                    valueDecimals: 1
+                }
+            }],
+        });
+
+        let chartRamUsedPercent = Highcharts.chart('ram_used_percent', {
+            chart: {
+                type: 'line'
+            },
+            title: {
+                text: 'RAM Used percent'
+            },
+            xAxis: {
+                title: {
+                    text: 'Time'
+                },
+                categories: ramUsedPercentTime,
+                tickInterval: 24 * 3600 * 1000
+            },
+            yAxis: {
+                title: {
+                    text: 'Used %'
+                },
+                max: 100
+            },
+            legend: {
+              enabled: false
+            },
+            colors: ['#ff3838'],
+            series: [{
+                name: 'RAM Used (%) - ' + deviceName,
+                data: ramUsedPercent,
+                tooltip: {
+                    valueDecimals: 1
+                }
+            }],
+        });
+    }
+
+
+    /**
+     * Selecting the time interval for displaying graph data.
+     */
+    $('.button-timespan').on('click', function () {
+        let URL = '/deviceStatus/getMetrics/' + deviceName + '?timespan=' + $(this).attr('data-timespan');
+        activeTimespan = URL;
+
+        $('.button-timespan').removeClass('button-selection');
+        $(this).addClass('button-selection');
+
+        sendJSONRequest(URL, null, RequestMethod.GET, beforeSendEmpty,
+                        renderDeviceStatusCharts, debug_callback, process_failures);
+    });
+
+    /**
+     *  Function reload charts. Depending on the selected time interval.
+     */
+    $('.button-device-status-refresh').click(function () {
+        sendJSONRequest(activeTimespan, null, RequestMethod.GET, beforeSendEmpty,
+                        renderDeviceStatusCharts, debug_callback, process_failures);
+    });
+
+    getDataDeviceStatus();
+
+    /**
+     * Function reload charts (auto). Depending on the selected time interval.
+     */
+    setTimeout(function () {
```

Comment:
setTimeout only calls the function once. Since we need periodic changes, we need setInterval here.